modules/tinymce/src/core/main/ts/api/dom/TreeWalker.ts

```diff
@@ -36,19 +36,19 @@ class DomTreeWalker {
    * Returns the current node.
    *
    * @method current
-   * @return {Node} Current node where the walker is.
+   * @return {Node | undefined} Current node where the walker is, or undefined if the walker has reached the end.
```

Comment:
This isn't the correct syntax for different doc types. It should be something like this:
```suggestion
   * @return {Node/undefined} Current node where the walker is, or undefined if the walker has reached the end.
```